brayns/Brayns.cpp

```diff
@@ -633,26 +608,18 @@ struct Brayns::Impl : public PluginAPI
 
     void _increaseAnimationFrame()
     {
-        if (_engine->getScene().getSimulationHandler() &&
-            !_engine->getScene().getSimulationHandler()->isReady())
-        {
-            return;
-        }
-
         auto& animParams = _parametersManager.getAnimationParameters();
+        if (!animParams.canUpdateFrame())
+            return;
         const auto animationFrame = animParams.getFrame();
         animParams.setFrame(animationFrame + 1);
     }
 
     void _decreaseAnimationFrame()
```

Comment:
Remove and use a -1 as the function parameter for the call suggested above.

Reply:
done